$NetBSD: patch-src_core_CMakeLists.txt,v 1.2 2019/10/29 19:30:11 bouyer Exp $

Add ApplicationServices framework
Use Q_WS_MAC to conditionalize frameworks
This sort-of matches Q_OS_MACX conditionals in main.cpp and qgisapp.cpp
but more importantly avoids using APPLE which we overwrite in the
pkgsrc Makefile to disable the MAC application bundle

--- src/core/CMakeLists.txt.orig	2019-09-13 14:11:09.000000000 +0200
+++ src/core/CMakeLists.txt	2019-10-27 19:11:40.694784656 +0100
@@ -1369,9 +1369,9 @@
   TARGET_LINK_LIBRARIES(qgis_core ${LIBTASN1_LIBRARY})
 ENDIF (APPLE)
 
-IF (APPLE AND NOT IOS)
+IF (Q_WS_MAC)
   TARGET_LINK_LIBRARIES(qgis_core qgis_native)
-ENDIF (APPLE AND NOT IOS)
+ENDIF (Q_WS_MAC)
 
 IF (QT_MOBILITY_LOCATION_FOUND)
   TARGET_LINK_LIBRARIES(qgis_core ${QT_MOBILITY_LOCATION_LIBRARY})
